Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,12 +1,18 @@
 #include "BatteryStateOfChargeService.h"
+#include "BatteryData.h"
+#include "QTime"
```

Comment:
Library headers should be in angled braces. Quotes are usually reserved for files in the project.
`<QTime>`